content/news/09-02-21-the-open-revolution-talk

```diff
@@ -0,0 +1,20 @@
+title: "The Open Revolution: Making a radically fairer and free future"
+created: 2021-02-09
+slug: the-open-revolution-making-a-radically-fairer-and-free-future
```

Comment:
slug is not needed - we can use the filename

in terms of filename / slug i'd recommend something like:

`talk-open-access-week-2021-open-revolution`

To keep it pithy and descriptive